docs/backend/testing/acceptance-testing-syncada-edi-invoicing.md

```diff
@@ -13,6 +13,10 @@ use.*
 
 This section will describe the prerequisites for invoicing, that are beyond the scope of documenting here. See the [References](#references) section for links that may help in setting these up.
 
+:::note
+The following commands listed assume you are in the `cmd` directory of the mymove project.
+:::
+
```

Comment:
If you run `make build_tools` it will build all these binaries in the local `bin/`directory. Then the `.envrc` file on line 11 adds the `bin/` directory to your path. I don't know if this note should say this, but perhaps say that you need to run `make build_tools` in order to get these tools.

[=> line 11 of the .envrc file](https://github.com/transcom/mymove/blob/master/.envrc#L11)

If we want to have only a single tool, since this is using `prime-api-client` and `generate-payment-request-edi` then we can have this note show `make bin/prime-api-client`.

```suggestion
:::note
The following commands listed assume you have built the local tooling required. You may either run `make build_tools` which will build all the tooling for the project. Or to run the commands in this document, run `make bin/prime-api-client generate-payment-request-edi`. This will allow you to run the commands without any prefixes.
:::

```